src/main/java/com/jcabi/github/RtStatus.java

```diff
@@ -42,7 +41,6 @@
  * @version $Id$
  * @since 0.23
  */
-@Immutable
```

Comment:
@cvrebert Why is this removed? I don't see any changes that impacts this class' immutability.